protocol/common/shard/shard.go

```diff
@@ -54,3 +54,10 @@ const UndefinedShardValue = 0
 func DefaultShardPubsubTopic() string {
 	return wakuproto.NewStaticShardingPubsubTopic(MainStatusShardCluster, DefaultShardIndex).String()
 }
+
+func DefaultShard() *Shard {
+	return &Shard{
+		Cluster: MainStatusShardCluster,
+		Index:   NonProtectedShardIndex,
```

Comment:
Why is NonProtectedShardIndex used here instead of the DefaultShardIndex?

Reply:
I saw that we put everything there (and my published shard info is public data), so decided to use this shard... can be changed. I didn't see we use DefaultShardIndex somewhere in the community.
Should I use DefaultShardIndex instead?

Reply:
The idea of the NonProtectedShardIndex is a shard used for users that have not joined a community to be able to send messages without having the DoS protection enabled. 
- ApplicationMetadataMessage_COMMUNITY_REQUEST_TO_JOIN
- ApplicationMetadataMessage_COMMUNITY_CANCEL_REQUEST_TO_JOIN
- ApplicationMetadataMessage_COMMUNITY_REQUEST_TO_JOIN_RESPONSE

Looking at code, it should be okay to keep it there for now, so no need to move it to the default shard. 
Probably in the future this type of control messages will be published into its own separate shard anyway.